bitnami/argo-cd/Chart.yaml

```
@@ -39,4 +39,4 @@ maintainers:
 name: argo-cd
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/argo-cd
-version: 6.4.8
+version: 6.4.9
```

As you are adding new parameters, we should release a new minor version 

```suggestion
version: 6.5.0
```